### PR TITLE
Make it embeddable

### PR DESCRIPTION
**Pull request by Gaurav0**
Fixes issue #13

Note: This builds on pull request #141, please review that first.

### CODE REVIEW

File: app/components/file-menu.js [hunk 1]
```diff
@@ -23,6 +23,12 @@ export default Ember.Component.extend({
     share() {
       prompt('Ctrl + C ;-)', window.location.href);
     },
+    embed() {
+      let src = window.location.href.split("?")[0];
+      src += "?numColumns=0";
+      let embedCode = `<iframe width="800" height="600" src="${src}"></iframe>`;
```
Comment (stefanpenner) [inline app/components/file-menu.js]: the resulting embed values are not correctly sanitized, and could pose a XSS risk.

vector:

1. evil twiddler sends cleverly crafted href, ble to escape the src + frame tag in the interpolated string.

Comment (Gaurav0) [inline app/components/file-menu.js]: That's really cool. But we are already using href in Share Twiddle. I'm going to open a PR ASAP and update Ember Twiddle for security reasons.

**Comment by stefanpenner** [comment]
i left comments re: the approach taking by columns [here](https://github.com/ember-cli/ember-twiddle/pull/141)

File: app/components/file-menu.js (hunk 1 above)
Comment (Gaurav0) [inline app/components/file-menu.js]: Actually maybe I'm misunderstanding. According to this, using window.location.href is relatively safe, and generally more safe than alternatives: http://stackoverflow.com/questions/24078332/is-it-secure-to-use-window-location-href-directly-without-validation

**Comment by Gaurav0** [comment]
@stefanpenner @rwjblue @joostdevries Please review.

**Comment by stefanpenner** [comment]
sgtm

**Pull request merged**